### PR TITLE
feat: rename module name

### PR DESCRIPTION
**Pull request by JalinWang**
fix https://github.com/casbin/gorm-adapter-ex/pull/1#issuecomment-1229482411

### CODE REVIEW

File: adapter.go [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package gormadapter
+package gorma_dapter_ex
```
Comment (hsluoyz) [inline adapter.go]: typo, and plz use `gormadapterex`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Go
 
 on:
   push:
-    branches: [ master ]
+    branches: []
```
Comment (hsluoyz) [inline .github/workflows/ci.yml:5]: Why delete?

File: adapter_test.go [hunk 3]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package gormadapter
+package gorma_dapter_ex
```
Comment (hsluoyz) [inline adapter_test.go]: typo, and plz use `gormadapterex`

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: adapter.go (hunk 1 above)
Comment (JalinWang) [reply adapter.go]: ok

File: .github/workflows/ci.yml (hunk 2 above)
Comment (JalinWang) [reply .github/workflows/ci.yml:5]: This enables to run test before creating PRs.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**